### ISSUE TITLE
tailwindcss not clean up whitespace when whitespace position on the first or last string

### ISSUE THREAD

**Issue by dstrygwyr** [biomejs/biome#6767]
### Environment information

```bash
tailwindcss not clean up whitespace when whitespace position on the first or last string
```

### Configuration

```JSON
{
  "$schema": "https://biomejs.dev/schemas/2.0.6/schema.json",
  "vcs": {
    "enabled": true,
    "clientKind": "git",
    "useIgnoreFile": true
  },
  "files": {
    "ignoreUnknown": false
  },
  "formatter": {
    "enabled": true,
    "indentStyle": "tab",
    "formatWithErrors": true,
    "indentWidth": 2,
    "attributePosition": "auto",
    "lineWidth": 80,
    "lineEnding": "lf",
    "bracketSpacing": true
  },
  "linter": {
    "enabled": true,
    "domains": {
      "next": "all"
    },
    "rules": {
      "recommended": false,
      "nursery": {
        "useSortedClasses": {
          "fix": "safe",
          "level": "info",
          "options": {
            "attributes": [
              "className"
            ],
            "functions": [
              "cn",
              "clsx",
              "classNames",
              "cva"
            ]
          }
        }
      },
      "performance": {
        "noImgElement": {
          "level": "warn"
        }
      },
      "correctness": {
        "noUnusedVariables": {
          "level": "error",
          "fix": "none",
          "options": {
            "ignoreRestSiblings": false
          }
        },
        "noUnusedImports": {
          "level": "error",
          "fix": "safe"
        },
        "useExhaustiveDependencies": {
          "fix": "none",
          "level": "info",
          "options": {
            "reportMissingDependenciesArray": true,
            "reportUnnecessaryDependencies": true
          }
        }
      }
    }
  },
  "javascript": {
    "formatter": {
      "quoteStyle": "double",
      "arrowParentheses": "always",
      "bracketSpacing": true,
      "jsxQuoteStyle": "double",
      "quoteProperties": "asNeeded",
      "semicolons": "always",
      "trailingCommas": "all"
    }
  },
  "json": {
    "parser": {
      "allowComments": true
    },
    "formatter": {
      "trailingCommas": "none"
    }
  },
  "assist": {
    "enabled": true,
    "actions": {
      "source": {
        "organizeImports": {
          "level": "on",
          "options": {
            "groups": [
              "react",
              "next/*",
              ":PACKAGE:",
              "@/hooks",
              "@/components",
              "@/utils",
              "@/lib",
              "@/assets"
            ]
          }
        }
      }
    }
  }
}
```

### Playground link

https://biomejs.dev/playground/?lintRules=useSortedClasses&code=PABwACAAYwBsAGEAcwBzAE4AYQBtAGUAPQAiAHQAZQB4AHQALQBzAG0AIABmAG8AbgB0AC0AYgBvAGwAZAAgACAAIAAgACAAIAAgACAAIAAgACAAIAAiAD4AaABlAGwAbABvACAAdwBvAHIAbABkADwALwBwAD4A

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
